week-1-practise/level-1/8-function-output-1.js

```diff
@@ -7,6 +7,10 @@ console.log(result);
 
 // Answer these questions:
 // - What does this program do?
+//generate random numbers everytime it's called, from 0 to one (not included)
```

Comment:
Are you sure it generates numbers in the range 0 to 1?

Hint: `Math.random` does but `getNumber` does not.

Reply:
I have updated them all ;D